assessment_software/src/app/dialogue-box/confirmationBox-component/confirmationBox.component.ts

```diff
@@ -0,0 +1,10 @@
+import {Component, Input} from '@angular/core';
+
+@Component({selector:'app-confirmation-box',
+templateUrl:'./confirmationBox.component.html',
+styleUrls:['./confirmationBox.component.css']})
+
+export class ConfirmationBoxComponent{
+    @Input() confirmation_message:string='Successfully submitted assignment';
+    @Input() confirmationStyle='confirm';
+}
```

Comment:
Currently, there is no way to remove the dialog when the X Icon is pressed at the top right corner. When the cancel button is pressed, the dialog should not be visible. It will need some research on the internet.